packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -455,12 +454,12 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Gesture
   @override
   void hitTest(HitTestResult result, Offset position) {
     assert(renderView != null);
```

Comment:
add asserts for result and position being non-null

Reply:
Is it necessary? I thought `--null-assertions` was added to avoid this kind of assert. /cc @jonahwilliams

Reply:
We can't make this the default because it breaks too much code

Reply:
Done